### PR TITLE
fix(gate): fixed the persistence of LegalEntityIdentifier

### PR DESCRIPTION
**Pull request by cezaralexandremorais**
fixed the persistence of the identifiers collection on the legal entity

### CODE REVIEW

File: bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+CREATE TABLE bpdmgate.legal_entity_identifiers (
```
Comment (nicoprow) [inline bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql:1]: Please remove schema names from migration files as this limits the flexibility on where these migration files can be applied to. Flyway handles on which schema this is executed

File: bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	value varchar(255) NOT NULL,
+	type_id varchar(255) NOT NULL,
+	legal_entity_id int8 NOT NULL,
+	issuing_body varchar(255) NULL,
+	CONSTRAINT pk_identifiers PRIMARY KEY (id),
+	CONSTRAINT uc_identifiers_uuid UNIQUE (uuid)
+);
+CREATE INDEX idx_9de08b456309ac30a77546592 ON bpdmgate.legal_entity_identifiers USING btree (legal_entity_id);
+
+ALTER TABLE bpdmgate.legal_entity_identifiers ADD CONSTRAINT fk_identifiers_on_partner FOREIGN KEY (legal_entity_id) REFERENCES bpdmgate.legal_entities(id);
```
Comment (nicoprow) [inline bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql:15]: also remove schema name here, see above

File: bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    created_at timestamp with time zone not null,
+    updated_at timestamp with time zone not null,
+	value varchar(255) NOT NULL,
+	type_id varchar(255) NOT NULL,
+	legal_entity_id int8 NOT NULL,
+	issuing_body varchar(255) NULL,
+	CONSTRAINT pk_identifiers PRIMARY KEY (id),
+	CONSTRAINT uc_identifiers_uuid UNIQUE (uuid)
+);
+CREATE INDEX idx_9de08b456309ac30a77546592 ON bpdmgate.legal_entity_identifiers USING btree (legal_entity_id);
```
Comment (nicoprow) [inline bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql:13]: remove schema name, see above

**Review by nicoprow — CHANGES_REQUESTED** [review]
Logic works for Legal Entities. Needs to have some changes to the migration file before it can be merged. Also the migration file's version number needs to incremented since there has come a new migration file meanwhile. Definitely needs a rebase on main.

Note: This is a solution for legal entity identifiers. There would be another pull request needed for the address identifiers.

File: bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql (hunk 1 above)
Comment (cezaralexandremorais) [reply bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql]: done

File: bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql (hunk 3 above)
Comment (cezaralexandremorais) [reply bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql]: done

**Review by nicoprow — APPROVED** [review]

**Pull request merged**